src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -89,4 +94,25 @@ private String createNextUrl(String jobId, int offset, int limit) {
 
     return sendQuery(querySql, dryRun);
   }
+
+  @Override
+  public ResponseEntity<QueryResponseData> uniqueValues(String version, @Valid String body) {
+
+    String bqTable = "gdc-bq-sample";
+    Map<String, String> tableParts = QueryTranslator.parseTableName(body);
```

Comment:
I think the body should be just the column name, not the table as well, as users shouldn't need to provide the table name.

Reply:
That is what I'm leaving it as now.